src/goto-instrument/full_slicer.cpp

```diff
@@ -322,18 +329,38 @@ Function: implicit
 
 \*******************************************************************/
 
-static bool implicit(goto_programt::const_targett target)
+static bool implicit(const namespacet &ns, goto_programt::const_targett target)
 {
   // some variables are used during symbolic execution only
 
+  const irep_idt &statement=target->code.get(ID_statement);
+  if (statement==ID_array_copy) return true;
+
+  if (target->is_assume()) return true;
```

Comment:
The control-flow dependency code is not handling this. Should we open a bug for this issue then?